locale/circuitpython.pot

```diff
@@ -35,6 +35,20 @@ msgid ""
 "https://github.com/adafruit/circuitpython/issues\n"
 msgstr ""
 
+#: main.c
+msgid ""
+"\n"
+"WARNING: Allocating pystack failed, defaulting back to build value.\n"
+"\n"
+msgstr ""
+
+#: main.c
+msgid ""
+"\n"
+"WARNING: Invalid CIRCUITPY_PYSTACK_SIZE, defaulting back to build value.\n"
+"\n"
+msgstr ""
+
```

Comment:
Strings take up a lot of space and I don't think these will be hit very often. I'd rather omit them or reuse an existing message.

Reply:
Did as discussed on discord. [Link](https://discord.com/channels/327254708534116352/327298996332658690/1075883666380701766) to nearby message.